client/components/ScoreContainer.js

```diff
@@ -161,7 +163,7 @@ export default function ScoreContainer({ type, data, isOwner }) {
     useTheme();
   const styles = useCustomStyles(loadStyles);
   const id = data._id;
-  const totalScore = data.totalScore;
+  const totalScore = computeTotalScore(data?.scores);
```

Comment:
This is going to break the pack scoring, let's use the virtual property you set up instead?